tests/cpp/predictor/test_gpu_predictor.cu

```diff
@@ -57,6 +57,72 @@ TEST(GPUPredictor, Basic) {
   }
 }
 
+namespace {
+void VerifyBasicColumnSplit(std::array<std::vector<float>, 32> const& expected_result) {
+  auto const world_size = collective::GetWorldSize();
+  auto const rank = collective::GetRank();
+
+  auto lparam = MakeCUDACtx(rank);
```

Comment:
Let's keep the naming `ctx`, easier for me to grep ;-)

Reply:
Done.